Enhance Cloudflare One Documentation Descriptions (#21799)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/applications/casb/troubleshooting/troubleshoot-integrations.mdx

  order: 1
---

> description: 'Learn how to troubleshoot Cloudflare CASB integrations, identify unhealthy or outdated integrations, repair issues, and upgrade for new features.'

Reviewer: ```suggestion
Cloudflare CASB detects when integrations are unhealthy or outdated.
```
